pages/common/az.md

```diff
@@ -27,3 +27,11 @@
 - Manage Azure Network resources:
 
 `az network`
+
+- Start in interactive mode:
```

Comment:
```suggestion
- Start in the interactive mode:
```

Reply:
This feels a bit archaic. Similar to if instead of "Display help", we'd say "Display the help"